Source/Scene/PolylineCollection.js

```diff
@@ -1049,19 +1049,29 @@ import SceneMode from './SceneMode.js';
     function removePolylines(collection) {
         if (collection._polylinesRemoved) {
             collection._polylinesRemoved = false;
+            var definedPolylines = [];
+            var polyIndex=0;
+            var polyline;
+            var i;
 
-            var polylines = [];
-
-            var length = collection._polylines.length;
```

Comment:
Style / performance: cesium code intentionally stores the length in a separate variable so that iteration is faster. Probably less important now that JS engines are much faster than in 2011 but it's a convention that has remained in the code.